### PR TITLE
WIP PAYT-423 Update payment gateway docs for v2

### PR DESCRIPTION
**Pull request by GeorgeF72**
Updates the payment gateway API and integration script docs for the new V2

This includes some changes to the integration guide to remove parts of the documentation that were never actually part of the live client script that was released.

### CODE REVIEW

File: site/payment-gateway/integration.md [hunk 1]
```diff
@@ -59,30 +59,28 @@ Here is a very simple example snippet that demonstrates this:
 ... any other content you have on your page
 
 <!-- Add the frontend payment script -->
-<script src="https://payment-gateway.holidayextras.co.uk/payment.1.0.0.min.js"></script>
+<script src="https://payment-gateway.holidayextras.co.uk/payment.2.0.0.min.js"></script>
```
Comment (shackpank) [inline site/payment-gateway/integration.md:62]: ✅ This... mostly... worked for me - the postMessage never appears but I think I can pin that on it deliberately specifying an environment-specific origin - the staging script won't reach a local `gabble.holidayextras.co.uk:5000`. But it is looking up the provided email.

File: site/payment-gateway/index.md [hunk 2]
```diff
@@ -12,14 +12,22 @@ Please read the integration guide before attempting to make any requests to thes
...
+| Make Payment | https://payment-gateway.holidayextras.co.uk/pay | POST | [v1](v1/pay) [v2](v2/pay) |
 
 ## Refund
 
-| Request | Endpoint | Method |
-| ------- | -------- | ------ |
-| Make Refund | [https://payment-gateway.holidayextras.co.uk/refund](refund) | POST |
+| Request | Endpoint | Method | Versions |
+| ------- | -------- | ------ | -------- |
+| Make Refund | https://payment-gateway.holidayextras.co.uk/refund | POST | [v1](v1/refund) [v2](v2/refund) |
```
Comment (shackpank) [inline site/payment-gateway/index.md:33]: Awaiting Tom's work before this can be tested

**Review by shackpank — COMMENTED** [review]
Have tested some of this out and all looking good so far

File: site/payment-gateway/v2/pay.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+NB: All parameter names are case sensitive.
+
+ | Name  | Data Type | Format | Mandatory? | Additional Information |
+ | ----  | --------- | ------ | ---------- | ---------------------- |
+ | channel | String | [A-Z] | Y | This will be supplied with your authentication credentials. |
+ | paymentId | String | [0-9A-Z]{16} | N | A unique ID to be used when making the payment. This cannot already exist otherwise payment will fail. If this is not supplied, then one will be generated for this payment and returned as part of the result. |
+ | token   | String | [0-9] | Y | A tokenised card to make the payment against. This needs to exist already, otherwise payment will fail. |
+ | paymentReference | String | [A-Z] | Y | A reference used to group multiple payments together, doesn't need to be unique. |
+ | amount | String | [0-9] | Y | The price (in pence) to be paid. |
```
Comment (shackpank) [inline site/payment-gateway/v2/pay.md:45]: This works as either a string or a number, the refund endpoint also documents a string but *requires* a number - these should both be swapped for numbers I think

**Review by tomnunn — APPROVED** [review]
Changes look good around `Pay`

**Pull request merged**